Src/BootCamp.Chapter/Account.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    public class Account
+    {
+        private readonly string _name;
+        private readonly decimal[] _balance;
+        private readonly string[] _accountArray;
+        private char _delimiter = ',';
+
+        public Account(string personAndBalance)
+        {
+            _accountArray = ArrayOps.ConvertToAccountArray(personAndBalance, _delimiter);
+            _name = AccountOps.GetNameForPerson(_accountArray);
+            _balance = AccountOps.GetBalanceForPerson(_accountArray);
+        }
+
+        public string GetName()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetName' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5054323)